content/webapp/components/ThemeSourcedDescription/index.tsx

```diff
@@ -0,0 +1,142 @@
+import { FunctionComponent, useEffect, useRef, useState } from 'react';
+import styled from 'styled-components';
+import Space from '@weco/common/views/components/styled/Space';
+import { font } from '@weco/common/utils/classnames';
+import WikidataLogo from '@weco/content/components/ThemeSourcedDescription/WikidataLogo';
+import MeshLogo from '@weco/content/components/ThemeSourcedDescription/mesh-logo.png';
+import Image from 'next/image';
```

Comment:
Saving this locally reorganises those for me ([it's an eslint rule we have](https://github.com/wellcomecollection/wellcomecollection.org/blob/main/.eslintrc.js#L20-L41)), could you double check your linting is set-up? It might be a missing extension, we use `dbaeumer.vscode-eslint`!